src/main_WorldCacheTest.cpp

```diff
@@ -0,0 +1,64 @@
+#include "BsFPSCamera.h"
+#include "REGothEngine.hpp"
+#include "components/AnchoredTextLabels.h"
+#include <BsZenLib/ImportPath.hpp>
+#include <Components/BsCCamera.h>
+#include <FileSystem/BsFileSystem.h>
+#include <GUI/BsCGUIWidget.h>
+#include <Resources/BsResources.h>
+#include <Scene/BsPrefab.h>
+#include <Scene/BsSceneObject.h>
+#include <Utility/BsTimer.h>
+#include <components/Item.hpp>
+#include <components/Waynet.hpp>
+#include <components/Waypoint.hpp>
+#include <daedalus/DATFile.h>
+#include <exception/Throw.hpp>
+#include <original-content/VirtualFileSystem.hpp>
+#include <components/GameWorld.hpp>
+
+class REGothWorldCacheTest : public REGoth::REGothEngine
+{
+public:
+  void setupMainCamera() override
+  {
+    REGoth::REGothEngine::setupMainCamera();
+
+    mMainCamera->SO()->addComponent<bs::FPSCamera>();
+  }
+
+  void setupScene() override
+  {
+    using namespace REGoth;
+
+    const bs::String world = "OLDWORLD.ZEN";
+    const bs::String saveGame = "MySafeGame";
+
+    bs::Timer timer;
```

Comment:
I assume you had this in here to see how much time it takes?
I think it is fine to actually use it her, since it is a TestApp.